modules/ROOT/pages/exception-filter.adoc

```diff
@@ -4,8 +4,55 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: error handling, exceptions, exception catching, exceptions, filter
 
-*Purpose:* Filters messages according to the type of an exception.
+The Exception Filter allows filtering of messages according to the type of an exception.
 
-*Minimum Configuration:* The class that indicates the type of the exception.
+Prerequisites: You need a Catch Exception Strategy configured in your flow.
 
-*Discussion:* An Exception Filter matches on the type of an exception. You supply the class indicating the exception type to the Expected Type property. For example, for a runtime exception type, you might set Expected Type to `java.lang.RuntimeException`. You can enter the class name or browse to it.
\ No newline at end of file
+Minimum Configuration: The class that indicates the type of the exception.
+
+An Exception Filter matches the type of exception you configure in the *Expected Type* field in the *Global Element* properties in Studio, or in the `expectedType` property in the Configuration XML.
+
+The Exception Filter expects an exception; this means that you need to add it within a Catch Exception Strategy processor. The exception is passed to the next processor in the exception handler only if it matches the Exception Filter's configured `expectedType`.
+
+Use the Exception Filter to decide if your Catch Exception Strategy needs to handle an exception or not, based on the type of the exception. When the Exception Filter receives an exception that does not match its configured `expectedType`, it returns a `null` value, and the processing of that message stops.
```

Comment:
"to decide if" -> "to check whether" or maybe "to indicate whether" 

reason: "decide" is a bit anthropomorphic